src/Acuminator/Acuminator.Utilities/Common/Constants.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Acuminator.Utilities.Roslyn.Semantic.Symbols
+{
+	public static class Constants
+	{
+		public static class Types
+		{
+
+			#region FieldAttributeSymbols
+
+			/// <summary>
+			/// The PXDBPackedIntegerArrayAttribute type full name in Acumatica 2018R2. Doesn't exist in previous versions.
+			/// </summary>
+			static readonly string PXDBPackedIntegerArrayAttributeFullName_Acumatica2018R2 = "PX.Data.PXDBPackedIntegerArrayAttribute";
```

Comment:
@saloponov  all code using Types does not compile because the default protection level when it is not specified explicitly is `private`. Also there is a common style convention in .Net to always specify protection level